ui/src/common/DateTimeSelector.tsx

```diff
@@ -9,22 +9,24 @@ interface DateTimeSelectorProps {
     showDate: boolean;
     label: string;
     popoverOpen?: (open: boolean) => void;
+    style?: object;
 }
```

Comment:
Text box label out of bounds

![image](https://github.com/user-attachments/assets/1c3a0b79-27da-4622-a977-430b71575857)

![image](https://github.com/user-attachments/assets/291d84a0-64c0-487c-b539-a3593911f611)

Reply:
Fixed the global css for this.